### PR TITLE
Add cfp

### LINKED ISSUE #25: Update website with CFP in FEB
drafted here: https://docs.google.com/document/d/14-N79UdKsayX995NPaNj4a2snVqs3aP2rFS2MfKoHVI/edit# 

final markdown available: https://hackmd.io/s/S1wc8p-OG#

### PR DESCRIPTION
**Pull request by dcwalk**
This resolves: https://github.com/ournetworks/2018/issues/25

Didn't go the single page route as it was burying the email sign-up. I think integrating this with #5 makes sense, but wasn't sure about where/what could be redirected given my proposed layout?

Can I get a quick review @garrying and @patcon? Hoping to distribute the CFP to networks Wednesday onward!!!

### CODE REVIEW

**Comment by patcon** [comment]
This looks great! I spent some time with GIMP and Inkscape trying to get a grayscape SVG of the DIWO heart so we could put it lightly in the background, but I just didn't have the wakefulness to figure it out right now :)

I know we're kinda past the copy phase, but seeing the very approachable amount of text on the front page, I'm wondering whether we can strip away some adjectives from this sentence, so that non-technical laypersons can more easily parse it:

> In the face of threats to the open access and resilience of our shared communications infrastructures, which tools and tactics will help us recognize the opportunities and challenges of this moment?

Without losing too much, could it become:

> In the face of threats to the open internet, which tools and tactics will help us recognize the opportunities and challenges of this moment?

**Comment by garrying** [comment]
The separate page route seems good to me. For #5, would it make sense for `/cfp` to redirect to `/2018`? A few thoughts that are not blockers to merging this for Wed:
- Header links: Would reversing the order, and lead with `2018`, make sense? They seem equally weighted at the moment and, related thought, I wonder if we should make a better distinction between one being the current CFP and the other being last year's edition either through labeling or moving the 2017 link to somewhere else on the page
- The Date/Title/Subtitle area seems a bit spaced out relative to the main text. Maybe we can left align and resolve the hierarchy. A sketch of what I'm thinking:
screen shot 2018-02-27 at 8 28 51 pm
- Should we have an explicit mention that this will be taking place in Toronto? Thinking through where this could get posted/shared

File: 2018.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+titleDisplay: "2018"
+date: 2018-07-13
+startTime: '9:00'
+endTime: '18:00'
+---
+
+<div style="text-align: center;height: 35vh;">
+  <span style="font-size: 6rem; font-weight: 600; line-height: 0.9;">July 13–18, 2018</span><br />
+  <span style="font-size: 4.25rem; font-weight: 600;">State of Our Networks</span><br />
+  <span style="font-size: 3.25rem; font-weight: 600;">Beyond DIY: Do it with others</span>
```
Comment (garrying) [inline 2018.md]: Should `Do it with others` be title case?

**Comment by dcwalk** [comment]
Thank you both for the review! I made the copy change on the first page @patcon and @garrying -- yr gonna hate me, but I inverted your sensible heading resolution because I'm SO INTO LARGE DATES rn. but also added the location... underneath. This could obviously be improved. 

I'm gonna merge in the interest of time, but we should obviously (and as we are hoping to within the next month) refine the whole site and this along with it

**Pull request merged**